pkg/resources/database_acceptance_test.go

```diff
@@ -8,43 +8,84 @@ import (
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 )
 
+func testCheckReplication(t *testing.T, path string, replicas, failovers []string) func(*terraform.State) error {
```

Comment:
Nit: Use uppercase for test `TestCheckReplication`

Reply:
Changed.